dbt/include/postgres/macros/adapters.sql

```diff
@@ -196,7 +196,12 @@
 
 {% macro postgres__alter_relation_comment(relation, comment) %}
   {% set escaped_comment = postgres_escape_comment(comment) %}
-  comment on {{ relation.type }} {{ relation }} is {{ escaped_comment }};
+  {% if relation.type = 'materialized_view' -%}
+    {% set relation_type = "materialized view" %}
+  {%- else -%}
+    {%- set relation_type = relation.type -%}
+  {%- endif -%}
+  comment on {{ relation_type }} {{ relation }} is {{ escaped_comment }};
```

Comment:
[tests are currently failing with below error](https://github.com/dbt-labs/dbt-postgres/actions/runs/9681773092/job/26719362444?pr=98#step:7:73417)

```py
ERROR tests/functional/unit_testing/test_ut_sources.py::TestUnitTestSourceInput::test_source_input - dbt_common.exceptions.base.CompilationError: Compilation Error
  expected token 'end of statement block', got '='
    line 3
      {% if relation.type = 'materialized_view' -%}
```

Reply:
This is now solved, all tests are passing.